test/e2e/testdata/zone-aware-routing-deployments.yaml

```diff
@@ -99,37 +99,3 @@ spec:
                       - proxy
               topologyKey: topology.kubernetes.io/zone
               namespaceSelector: {}
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: zone-aware-backend
-  namespace: gateway-conformance-infra
-  annotations:
-    service.kubernetes.io/topology-mode: Auto
```

Comment:
shouldnt we test both ?

Reply:
topology aware routing is still being tested I've just renamed the service and created a second for zone aware routing be enabled from the BackendTrafficPolicy.

Reply:
thanks